customizations/README.md

```diff
@@ -0,0 +1,187 @@
+
+
+## Access to CARTO from outside the cluster
+By default, the `router` Service of CARTO selfhosted is configured in mode `ClusterIP` so it's only usable from inside the cluster.
+To access to it you can exec locally a [kubectl port-forward](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) but this only make it accessible to your machine.
+
+To made it accessible from the internet (or outside the Kubernetes network) you need to properly configure the `router` Service (the easiest way) or create an Ingress.
+
+### General notes
+<!--
+TODO: Document timeout increment and disable internal TLS and so on
+TODO: Talk about static IP
+-->
+
+### Service
+
+That's the easiest way of open your CARTO selfhosted to the world.
+You need to change the `router` Service type to `LoadBalancer`.
+
+You can check an example [here](service_loadBalancer/config.yaml) (keep in mind the [general notes](#general-notes) before to proceed).
+
+<!--
+TODO: We need to talk about TLS and so on...
+-->
+
+We have examples for multiple cloud providers:
+- [AWS EKS](service_loadBalancer/aws_eks/config.yaml)
+<!--
+TODO: Add the other providers
+-->
+
+
+### Ingress
+<!--
+TODO: Document Ingress
+-->
+
+### Configure HTTPS
+By default, the package generate a self-signed certificate with a validity of 365 days.
+Some times you need to use a valid certificate or need to totally disable it to leave the management to an external proxy.
+
+⚠️ CARTO selfhosted only works if the final client use HTTPS protocol. ⚠️
+
+#### Disable internal HTTPS
+<!--
+TODO: Document and add the ability to do it
+-->
+
+#### Configure your own certificates in CARTO
+
+- Create a kubernetes secret with following content:
+  ```bash
+  kubectl create secret tls -n <namespace> <your_own_installation_name|carto>-tls-certificate --cert=path/to/cert/file --key=path/to/key/file
+  ```
+
+- [Add the following customization](#how-to-define-customizations) lines:
+
+  ```yaml
+  tlsCerts:
+    autoGenerate: false
+    existingSecret:
+      name: "<your_own_installation_name|carto>-tls-certificate"
+      keyKey: "tls.key"
+      certKey: "tls.crt"
+  ```
+
+## Use external databases
+This package comes with an internal Postgres and Redis but it is not recommended for production. It does not have any logic for backups or any other monitoring. 
+
+So we recommend to use external databases, preferible managed database by your provider, with backups, high availability, etc.
+
+### Configure your own postgres
+CARTO selfhosted require a Postges (13+) to work.
+In that Postgres, CARTO stores some metadata and also the credentials of the external connections configured by the CARTO selfhosted users.
+
+⚠️ That Postgres has nothing to do with the connections that the user configures in the CARTO workspace since it stores the metadata of the entire CARTO selfhosted. ⚠️
+
+To proceed you need:
+- Create one secret in kubernetes with the Postgres passwords:
```

Comment:
I think we could add anothers comments to use the external postgres and redis without create the secret manually, using [externaldb-secret.yaml](https://github.com/CartoDB/carto-selfhosted-helm/blob/main/chart/templates/externaldb-secret.yaml) and [externalredis-secret.yaml](https://github.com/CartoDB/carto-selfhosted-helm/blob/main/chart/templates/externalredis-secret.yaml)

Reply:
I only copy/pasted and little bit adapted from the old config. I think that this section should be rethinked/redo